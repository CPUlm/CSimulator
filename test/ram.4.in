3
00
1
00  
1110
01 
1
00
0010
00
0
11
1111
