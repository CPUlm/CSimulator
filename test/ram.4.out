Step 1:
=> o = 0000
Step 2:
=> o = 0000
Step 3:
=> o = 0010
